diff --git a/ports/stm32/boards/NUCLEO_WB55/mpconfigboard.h b/ports/stm32/boards/NUCLEO_WB55/mpconfigboard.h
index 206134941..54911a0b9 100644
--- a/ports/stm32/boards/NUCLEO_WB55/mpconfigboard.h
+++ b/ports/stm32/boards/NUCLEO_WB55/mpconfigboard.h
@@ -5,7 +5,21 @@
 #define MICROPY_HW_BOARD_NAME       "NUCLEO-WB55"
 #define MICROPY_HW_MCU_NAME         "STM32WB55RGV6"
 
-#define MICROPY_PY_PYB_LEGACY       (0)
+#define MICROPY_PY_PYB_LEGACY               (0)
+#define MICROPY_PY_THREAD                   (0)
+#define MICROPY_PY_FRAMEBUF                 (0)
+#define MICROPY_PY_USOCKET                  (0)
+#define MICROPY_PY_NETWORK                  (0)
+#define MICROPY_PY_UHASHLIB                 (0)
+#define MICROPY_PY_UZLIB                    (0)
+#define MICROPY_PY_UHEAPQ                   (0)
+#define MICROPY_PY_UTIMEQ                   (0)
+#define MICROPY_PY_UASYNCIO                 (0)
+#define MICROPY_PY_URE                      (0)
+#define MICROPY_PY_URE_SUB                  (0)
+#define MICROPY_PY_UCTYPES                  (0)
+#define MICROPY_PY_BUILTINS_HELP_TEXT       mp_help_default_text
+#define MICROPY_PY_MICROPYTHON_HEAP_LOCKED  (1)
 
 #define MICROPY_HW_HAS_FLASH        (1)
 #define MICROPY_HW_ENABLE_RTC       (1)
