diff --git a/ports/stm32/Makefile b/ports/stm32/Makefile
index 61ce06ca8..96213bf19 100644
--- a/ports/stm32/Makefile
+++ b/ports/stm32/Makefile
@@ -350,6 +350,14 @@ SRC_HAL = $(addprefix $(HAL_DIR)/Src/stm32$(MCU_SERIES)xx_,\
 	hal_uart.c \
 	)
 
+ifeq ($(MCU_SERIES),$(filter $(MCU_SERIES),wb))
+SRC_HAL += $(addprefix $(HAL_DIR)/Src/stm32$(MCU_SERIES)xx_,\
+	hal_cryp.c \
+	hal_cryp_ex.c \
+	hal_pka.c \
+	)
+endif
+
 ifeq ($(MCU_SERIES),$(filter $(MCU_SERIES),f4 f7 h7 l0 l4 wb))
 SRC_HAL += $(addprefix $(HAL_DIR)/Src/stm32$(MCU_SERIES)xx_,\
 	ll_usb.c \
