diff --git a/ports/stm32/boards/NUCLEO_WB55/mpconfigboard.h b/ports/stm32/boards/NUCLEO_WB55/mpconfigboard.h
index 206134941..e7750dd0c 100644
--- a/ports/stm32/boards/NUCLEO_WB55/mpconfigboard.h
+++ b/ports/stm32/boards/NUCLEO_WB55/mpconfigboard.h
@@ -6,6 +6,11 @@
 #define MICROPY_HW_MCU_NAME         "STM32WB55RGV6"
 
 #define MICROPY_PY_PYB_LEGACY       (0)
+#define MICROPY_PY_THREAD           (0)
+#define MICROPY_PY_FRAMEBUF         (0)
+#define MICROPY_PY_USOCKET          (0)
+#define MICROPY_PY_NETWORK          (0)
+#define MICROPY_PY_UHASHLIB         (0)
 
 #define MICROPY_HW_HAS_FLASH        (1)
 #define MICROPY_HW_ENABLE_RTC       (1)
