diff --git a/extmod/extmod.mk b/extmod/extmod.mk
index ff24a549a..101f42875 100644
--- a/extmod/extmod.mk
+++ b/extmod/extmod.mk
@@ -104,7 +104,6 @@ SRC_MOD += $(addprefix $(MBEDTLS_DIR)/library/,\
 	md4.c \
 	md5.c \
 	md.c \
-	md_wrap.c \
 	oid.c \
 	padlock.c \
 	pem.c \
@@ -128,6 +127,7 @@ SRC_MOD += $(addprefix $(MBEDTLS_DIR)/library/,\
 	ssl_ciphersuites.c \
 	ssl_cli.c \
 	ssl_cookie.c \
+	ssl_msg.c \
 	ssl_srv.c \
 	ssl_ticket.c \
 	ssl_tls.c \
